src/main/java/frc/robot/DriveSystem.java

```diff
@@ -5,56 +5,42 @@
 public class DriveSystem {
     private double _driveCmd;
     private double _turnCmd;
-
+    //Initialization of variables
     private Talon _motorFL;
     private Talon _motorFR;
     private Talon _motorRL;
     private Talon _motorRR;
 
-    /**
-     * Default constructor
-     * 
-     * Instantiates the motor Talon objects with their correct channels and inversion state based
-     * on constants contained in the RobotConstants class, e.g. -
-     * 
-     *      MOTOR_CHANNEL_FL, MOTOR_CHANNEL_FR, ...
-     *      MOTOR_INVERTED_FL, MOTOR_INVERTED_FR, ...
-     * 
-     * Also sets the commands to 0.0 by default.
-     */
+    //MOTOR_CHANNEL_FL, MOTOR_CHANNEL_FR, ...
+    //MOTOR_INVERTED_FL, MOTOR_INVERTED_FR, ...
     public DriveSystem() {
+        this._motorFL = new Talon(RobotConstants.MOTOR_CHANNEL_FL);
+        this._motorFR = new Talon(RobotConstants.MOTOR_CHANNEL_FR);
+        this._motorRL = new Talon(RobotConstants.MOTOR_INVERTED_FL);
+        this._motorRR = new Talon(RobotConstants.MOTOR_INVERTED_RR);
 
+        this.reset();
```

Comment:
A manner of personal preference: I like to have member variables explicitly initialized in the constructor. You calling the `reset` function achieves the same effect implicitly. Changing it to an explicit initialization is not required to pass the PR, but I want you to be aware of the stylistic choice.